scripts/create-k8s.sh

```diff
@@ -27,7 +27,7 @@ sleep 600
 echo "Now setting up your k8s cluster ..."
 
 if [ "${k8s_provider}" == "eks" ]; then
-    cd ../../../ingress/
+    cd ../ingress/
```

Comment:
looks like this will fix a bug that would prevent a future cluster deployment, nice catch

Reply:
yes I am testing the create-k8s.sh against fuel-deployment and make modifications in this PR